src/typescript/frontend/src/components/pages/metadata/MetadataPage.tsx

```diff
@@ -1,35 +1,53 @@
 "use client";
 
 import { useAptos } from "context/wallet-context/AptosContextProvider";
-import { ChangeEventHandler, MouseEventHandler, useState } from "react";
+import { type ChangeEventHandler, type MouseEventHandler, useState } from "react";
 
-type MetadataField = { name: string, placeholder: string };
+type MetadataField = { name: string; placeholder: string };
 
-const DEFAULT_FIELDS: [MetadataField, string][] = [[{ name: "CTO link", placeholder: "https://..." }, ""], [{ name: "Moto", placeholder: "The best shoe market ever !" }, ""]];
+const DEFAULT_FIELDS: [MetadataField, string][] = [
+  [{ name: "CTO link", placeholder: "https://..." }, ""],
+  // cspell:word moto
+  [{ name: "Moto", placeholder: "The best shoe market ever!" }, ""],
```

Comment:
@CRBl69 Note the cspell ignore, not sure you meant to type this or not, not exactly sure what it means 😆